lgc/builder/llpcYCbCrAddressHandler.h

```diff
@@ -60,30 +60,31 @@ class YCbCrAddressHandler {
   // Get specific plane
   llvm::Value *getPlane(unsigned idx) {
     assert(idx < 3);
-    return planeBaseAddresses[idx];
+    return m_planeBaseAddresses[idx];
   }
 
   // Get pitch for Y plane
-  llvm::Value *getPitchY() { return pPitchY; }
+  llvm::Value *getPitchY() { return m_pitchY; }
 
   // Get pitch for Cb plane
-  llvm::Value *getPitchCb() { return pPitchCb; }
+  llvm::Value *getPitchCb() { return m_pitchCb; }
 
   // Get height for Y plane
-  llvm::Value *getHeightY() { return pHeightY; }
+  llvm::Value *getHeightY() { return m_heightY; }
 
   // Get Height for Cb plane
-  llvm::Value *getHeightCb() { return pHeightCb; }
+  llvm::Value *getHeightCb() { return m_heightCb; }
 
-  SqImgRsrcRegHandler *pRegHelper;
-  Builder *pBuilder;
-  llvm::SmallVector<llvm::Value *, 3> planeBaseAddresses;
-  llvm::Value *pPitchY;
-  llvm::Value *pHeightY;
-  llvm::Value *pPitchCb;
-  llvm::Value *pHeightCb;
-  llvm::Value *pOne;
-  GfxIpVersion *pGfxIp;
+private:
+  SqImgRsrcRegHandler *m_regHelper;
+  Builder *m_builder;
+  llvm::SmallVector<llvm::Value *, 3> m_planeBaseAddresses;
+  llvm::Value *m_pitchY;
```

Comment:
m_pitchY, m_heightY, m_pitchCb, m_heightCb are not initialize to nullptr. Please help to add the initialization to constructor